.azurepipelines/templates/CargoToolRepublish.yml

```diff
@@ -0,0 +1,194 @@
+## @file
+# Azure Pipeline to build and publish cargo tools in a format supported by edk2-pytool-extension's external dependency.
+#
+# This template will clone the repository specified in the `repo_url` parameter at the tag specified in the `tag`
+# parameter and then build the tool for X64 and ARM64 for Windows and/or linux. These binaries will be published as a
+# nuget package in an internal format supported by edk2-pytool-extensions.
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+parameters:
+  - name: tag
+    displayName: 'Cargo Make Version Tag'
+    type: string
+  - name: repo_url
+    displayName: 'Cargo Tool Repository URL'
+  - name: tool_name
+    displayName: 'Tool name (and nuget config file name)'
+    type: string
+  - name: extra_build_args
+    displayName: 'Extra build arguments'
+    type: string
+    default: ''
+  - name: upload_linux
+    displayName: 'Upload Linux Binaries (X64 and ARM64)'
+    type: boolean
+    default: true
+  - name: upload_windows
+    displayName: 'Upload Windows Binaries (X64 and ARM64)'
+    type: boolean
+    default: true
+  - name: python_version
+    displayName: 'Python Version'
+    type: string
+    default: '3.11'
+
+variables:
+  - name: windows_image
+    value: 'windows-latest'
+  - name: linux_image
+    value: 'ubuntu-22.04'
```

Comment:
Please use Ubuntu 24.04 since we've updated to that everywhere else.

Reply:
switched to 24.04